saml/saml.go

```diff
@@ -68,7 +68,7 @@ func extractArns(attrs []saml.Attribute) (arns []ARN) {
 				// 1. arn:aws:iam::xxxxxxxxxxxx:role/MyRole,arn:aws:iam::xxxxxxxxxxxx:saml-provider/MyProvider
 				// 2. arn:aws:iam::xxxxxxxxxxxx:saml-provider/MyProvider,arn:aws:iam::xxxxxxxxxxxx:role/MyRole
 				// Error otherwise.
-				components := strings.Split(av.Value, ",")
+				components := strings.Split(strings.TrimSpace(av.Value), ",")
```

Comment:
@johananl do you have a nice go-like way to actually trim each individual component without bloating the code?

Reply:
@allcloud-jonathan I apologize for the late response.
If you mean - "I want to trim each item in `components`", I think this would be the canonical Go way of doing this:

```go
components := []string{}
for _, i := range strings.Split(strings.TrimSpace(av.Value), ",") {
	components = append(components, strings.TrimSpace(i))
}
```